FAQ.md

```diff
@@ -199,9 +199,10 @@ Firecracker process remains unaware of the guest shutdown so it lives on.
 Running the `reboot` command in a Linux guest will gracefully bring down the guest
 system and also bring a graceful end to the Firecracker process.
 
-Issuing a `SendCtrlAltDel` action command through the Firecracker API will generate a
-`Ctrl + Alt + Del` keyboard event in the guest resulting in a clean reboot on most
-guest Linux systems.
+On `x86_64` systems, issuing a `SendCtrlAltDel` action command through the
+Firecracker API will generate a `Ctrl + Alt + Del` keyboard event in the guest
+resulting in a clean reboot on most guest Linux systems. This is, however, not
```

Comment:
I would also change reboot to shutdown.

Reply:
Isn't `SendCtrlAltDel` supposed to *reboot* the VM (as previously stated in the [FAQ](https://github.com/firecracker-microvm/firecracker/blob/master/FAQ.md#how-can-i-gracefully-reboot-the-guest-how-can-i-gracefully-poweroff-the-guest) prior to this PR)?

Reply:
In order to be really accurate, I would change this section to say that the "SendCtrlAltDel will generate a CtrlALtDel keyboard event which triggers a behavior identical to the one triggered when running the "reboot" command in the guest". This should be pretty clear since the perks of the "reboot" command are already detailed in the previous section. WDYT?

Reply:
Updated it per your suggestion, PTAL!